server/etcdserver/api/membership/cluster.go

```diff
@@ -305,7 +305,13 @@ func (c *RaftCluster) Recover(onSet func(*zap.Logger, *semver.Version)) {
 // ensures that it is still valid.
 func (c *RaftCluster) ValidateConfigurationChange(cc raftpb.ConfChange) error {
 	// TODO: this must be switched to backend as well.
-	membersMap, removedMap := membersFromStore(c.lg, c.v2store)
+	var membersMap map[types.ID]*Member
+	var removedMap map[types.ID]bool
+	if c.v2store != nil {
+		membersMap, removedMap = membersFromStore(c.lg, c.v2store)
+	} else {
+		membersMap, removedMap = c.be.MustReadMembersFromBackend()
+	}
```

Comment:
This doesn't seem like a test change but as a logic change to validate config change based on v3 store if v2store is not around. FYI since v3.5 v2 and v3 stores should include membership information, but the v3store membership should be authoritative. 

Please be clear on what this Pr is meant to do. To add tests or  to move validation to v3 store.

Reply:
As of now, this PR is meant for testing ConfChange entries with membership info from be. It does not change logic used by production.
In the code snip above, the else path is inactive in production since v2store is set. Ref:https://github.com/etcd-io/etcd/blob/main/server/etcdserver/bootstrap.go#L449

The test will take the else path and exercise ConfChange validation using info from be.

Reply:
Please don't mix adding tests and changing logic. Fact that code you are adding will not change the fact that it adds logic.

Reply:
The test cannot be added without the change in cluster.go discussed above. Can you please clarify how might I just add the test in this case?

Reply:
@serathius can you please comment on this? should this test not be added at this time?

Reply:
I don't understand the change proposed. We  should be already migrated to read membership from be as v3.5 was meant to do that.